docs/scenes/get-started/set-up-your-environment/SelectCard.tsx

```diff
@@ -0,0 +1,76 @@
+import { ButtonBase, mergeClasses, useTheme } from '@expo/styleguide';
+import { useEffect, useState } from 'react';
+
+import { CALLOUT, HEADLINE } from '~/ui/components/Text';
+
+type Props = {
+  imgSrc: string;
+  darkImgSrc?: string;
+  alt: string;
+  title: string;
+  description?: string;
+  isSelected: boolean;
+  onClick: () => void;
+};
+
+export function SelectCard({
+  imgSrc,
+  darkImgSrc,
+  alt,
+  title,
+  description,
+  isSelected,
+  onClick,
+}: Props) {
+  const { themeName } = useTheme();
+  const [isDarkMode, setIsDarkMode] = useState(false);
+
+  useEffect(
+    function didMount() {
+      const darkMode =
+        themeName === 'dark' ||
+        (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches);
+      setIsDarkMode(darkMode);
+    },
+    [themeName]
+  );
+
+  return (
+    <ButtonBase onClick={onClick}>
+      <div className="w-[250px] shadow-xs overflow-hidden rounded-lg flex flex-col items-center border border-default hocus:scale-[102%] transition-all hocus:shadow-sm">
+        <div
+          className={mergeClasses(
+            'border-b border-default',
+            isSelected ? 'bg-gradient-to-b from-palette-blue3 to-palette-blue4' : 'bg-subtle'
+          )}>
+          <picture className="relative w-auto h-full flex items-end">
+            {isDarkMode && <source srcSet={darkImgSrc} type="image/png" />}
+            <img
+              src={imgSrc}
+              alt={alt}
+              className={mergeClasses(isSelected ? 'grayscale-0' : 'grayscale opacity-80')}
+            />
+          </picture>
+        </div>
+        <div className="p-3 text-center flex items-center justify-center flex-col gap-2">
+          <HEADLINE className="text-center">{title}</HEADLINE>
+          {description ? <CALLOUT theme="secondary">{description}</CALLOUT> : null}
+          <div className="relative">
+            <div
+              className={mergeClasses(
+                'size-5 rounded-full border bg-default',
+                isSelected ? 'border-palette-blue9' : 'border-default'
+              )}
+            />
+            <div
+              className={mergeClasses(
+                'size-3 rounded-full absolute top-1 right-1',
+                isSelected ? 'border-palette-blue9 bg-palette-blue9' : 'bg-transparent'
+              )}
+            />
+          </div>
```

Comment:
Let's move in `Radio` input component, instead of trying to recreate only the visual part of it. 

I really don't like the radio placement and alignment of the tiles content. Can we move the input indicator to the header line and stick to the text aligned to the left? Also the interaction animation is not exactly what we usually use, would be nice if we can move scale to only apply to asset used in tile, not the whole tile.

<img width="698" alt="Screenshot 2024-05-07 103250" src="https://github.com/expo/expo/assets/719641/140ad8c0-e048-4202-9315-3fa3f8e8405c">

Reply:
I am currently about to be on a plane, and this needs to go out today. With that, I can address this in about 12 hours in another PR. I understand that we want to change the design/functionality here. I will get that PR up later today.